tekdoc/main.go

```diff
@@ -17,15 +17,100 @@ limitations under the License.
 package main
 
 import (
+	"flag"
 	"fmt"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned/scheme"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"text/template"
+)
 
-	_ "github.com/tektoncd/pipeline/pkg/pod"
+var (
+	filename = flag.String("f", "", "Name of the file to parse")
 )
 
+// TODO: take a file path, parse it as YAML -> v1alpha1.Task
+
+func read(name string) (v1alpha1.Task, error) {
```

Comment:
Probably want this function name to be a little more descriptive so it's easier to understand exactly what it's doing. `readTask` perhaps?